config/packages/security.yaml

```diff
@@ -1,5 +1,5 @@
 security:
-    always_authenticate_before_granting: true
+    enable_authenticator_manager: true
```

Comment:
Yes, this config is only for new projects, this configuration is not on the packages, so it will be in the symfony standard too when it'll be ready here.